### PR TITLE
feat: added some duration cel funcs with tests

### PR DESCRIPTION
**Pull request by adityathebe**

### CODE REVIEW

File: strings/duration_gen.go [hunk 1]
```diff
@@ -1,105 +1,42 @@
...
-	cel.Overload("duration.Minutes_",
-		nil,
-		cel.DoubleType,
-		cel.FunctionBinding(func(args ...ref.Val) ref.Val {
-
-			var x Duration
-
-			return types.DefaultTypeAdapter.NativeToValue(x.Minutes())
-
+var durationDurationGen = cel.Function("Duration",
```
Comment (adityathebe) [inline strings/duration_gen.go:30]: @moshloop The native cel `duration` func doesn't support days. so I had to add our own `Duration` func back.

File: .golangci.yml
Comment (adityathebe) [inline .golangci.yml:1]: disabled these two linters for now. Needs quite a bit of refactoring otherwise.

**Comment by moshloop** [comment]
@adityathebe can we add a few tests for functions here: https://github.com/google/cel-go/blob/master/ext

**Comment by adityathebe** [comment]
@moshloop Added extensions and tests as well. The latest cel-go version has more extensions (list, set) that're not available in the version we're using

**Comment by moshloop** [comment]
@adityathebe can we not upgrade the cel version ?

**Pull request merged**